sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Demo/Program.cs

```diff
@@ -33,5 +38,52 @@
     return $"Hello World! OpenTelemetry Trace: {Activity.Current?.Id}";
 });
 
+#if !NETFRAMEWORK
```

Comment:
Do you want to keep the stress in the basic demo project?

Reply:
No. I mentioned in the PR description that this must be reverted before merging. 
I committed this so reviewers can see how I've been doing my testing. I also needed to save a copy of this because we've been iterating frequently and I needed to reuse the stress test.

Reply:
to avoid the confusion keep this changes locally in your dev env.